File: src/main/java/org/vors/pairbot/service/GameService.java
package org.vors.pairbot.service;

import org.springframework.stereotype.Component;
import org.vors.pairbot.model.UserInfo;

@Component
public class GameService {

    public String xpShort(UserInfo user){
        return String.format("%dxp", user.getXp());
    }
}

File: src/main/java/org/vors/pairbot/service/GameService.kt
package org.vors.pairbot.service

import org.springframework.stereotype.Component
import org.vors.pairbot.model.UserInfo

@Component
class GameService {

    fun xpShort(user: UserInfo): String{
        return String.format("%dxp", user.xp)
    }
}
